Update List of features for 4.7 (#12036)
godotengine/godot-docs: about/list_of_features.rst

  rendering pipeline.
- Performance monitoring tools, including
  :ref:`custom performance monitors <doc_custom_performance_monitors>`.
- Supports :ref:`tracing profilers <doc_tracing_profilers>` like
  :ref:`doc_profiler_tracy` and :ref:`doc_profiler_perfetto` for deeper optimization tasks.
- Any script can be :ref:`run in the editor <doc_running_code_in_the_editor>`
  and provide custom functionality such as clickable buttons in the inspector,
  without needing to create an editor plugin.
- Live script reloading.
- Live scene editing.

   - Changes *will* reflect in the editor and *will* be kept after closing the running project.

- Live camera replication (disabled by default).

   - Move the in-editor camera and see the result in the running project.

- Remote inspector.

   - Changes *won't* reflect in the editor and *won't* be kept after closing the running project.

- Optional :ref:`game embedding <doc_game_embedding>` to run the project in a panel within the editor.

  - Select 2D and 3D nodes in the project's viewport to inspect them in the editor.
  - Move the camera within the project using the 2D and 3D camera overrides.

>   - Supports time scale adjustments, pausing and frame advance.

Reviewer: ```suggestion
  - Supports time scale adjustments, pausing, and frame advance.
```

3D graphics
-----------

> - HDR rendering in linear space (Forward+/Mobile), or SDR rendering in sRGB space

Reviewer 2: “Rendering” is a pretty vague term, so this ends up being somewhat misleading because it may mean lighting simulations or blending, which behave differently. But I think the intent with this like was to simply mean “3D lighting simulations”, which are always performed on linear-encoded values in all renderers, regardless of HDR 2D setting (when the glow effect is not considered a lighting simulation).

So I think this is probably better as something like this:

```suggestion
- Linear HDR internal lighting calculations.
```

I don’t think a clarification that these are RGB is needed (by not stating it’s spectral rendering, it’s implied RGB)

Reviewer 2: Then again, there’s a whole section to lighting that follows this, so maybe that’s a bad suggestion. I’m really not sure what this original bullet point was trying to convey; maybe better to move my suggestion to the lighting section and remove this point entirely…

**Real-time lighting:**

- :ref:`Directional lights <doc_lights_and_shadows_directional_light>` (sun/moon).

> - :ref:`Omnidirectional lights <doc_lights_and_shadows_omni_light>` .

Reviewer: ```suggestion
- :ref:`Omnidirectional lights <doc_lights_and_shadows_omni_light>`.
```

- Support for :ref:`rendering 3D at a lower resolution <doc_resolution_scaling>`
  while keeping 2D rendering at the original scale. This can be used to improve
  performance on low-end systems or improve visuals on high-end systems.
- Resolution scaling uses nearest-neighbor filtering, bilinear filtering,

>   AMD FidelityFX Super Resolution 1.0 (FSR1) or AMD FidelityFX Super Resolution 2.2.1

Reviewer: ```suggestion
  AMD FidelityFX Super Resolution 1.0 (FSR1), or AMD FidelityFX Super Resolution 2.2.1
```

- Support for re-routable :ref:`audio buses <doc_audio_buses>` and effects
  with dozens of effects included.
- Support for polyphony (playing several sounds from a single :ref:`class_AudioStreamPlayer`
  node).
- Support for random volume and pitch.
- Support for real-time pitch scaling.
- Support for sequential/random sample selection, including repetition prevention
  when using random sample selection.

> - :ref:`class_Listener2D` and :ref:`class_Listener3D` nodes to listen from a position

Reviewer: ```suggestion
- :ref:`class_AudioListener2D` and :ref:`class_AudioListener3D` nodes to listen from a position
```

- Support for re-routable :ref:`audio buses <doc_audio_buses>` and effects
  with dozens of effects included.
- Support for polyphony (playing several sounds from a single :ref:`class_AudioStreamPlayer`
  node).
- Support for random volume and pitch.
- Support for real-time pitch scaling.
- Support for sequential/random sample selection, including repetition prevention
  when using random sample selection.
- :ref:`class_AudioListener2D` and :ref:`class_AudioListener3D` nodes to listen from a position

>   different than the camera.

Reviewer: ```suggestion
  different from the camera.
```

- Buttons.
- Checkboxes, check buttons, radio buttons.
- Text entry using :ref:`class_LineEdit` (single line), :ref:`class_TextEdit`
  (multiple lines), and :ref:`class_CodeEdit` (supports syntax highlighting,
  line numbers, and more).
- Dropdown menus using :ref:`class_PopupMenu` and :ref:`class_OptionButton`
  with support for an optional search bar.
- Scrollbars.
- Labels.
- RichTextLabel for :ref:`text formatted using BBCode <doc_bbcode_in_richtextlabel>`,
  with support for animated custom effects.
- Trees (can also be used to represent tables).

> - Color picker with RGB, HSV, OKHSL modes and custom color palettes.

Reviewer: ```suggestion
- Color picker with RGB, HSV, and OKHSL modes and custom color palettes.
```
